dayliopy/clean_csv.py

```diff
@@ -8,7 +8,7 @@
 import csv
 import datetime
 
-from .lib.config import AppConf
+from lib.config import AppConf
```

Comment:
Make sure to run the app as a module. 

If you do `python app.py` then the relative imports with the dot will fail.

But if you do `python -m app` then it will work.

```suggestion
from .lib.config import AppConf
```

Reply:
From the repo root:

```sh
python -m dayliopy.clean_csv
```

See:

https://github.com/MichaelCurrin/daylio-csv-parser/blob/e0f7e4f489b8b3177990743fade5e1b194dd35a9/Makefile#L52

Reply:
I didn't know you could do that, thanks!